### PR TITLE
fix: move ibm-platform-services package to @ibm-cloud npm org

### PR DESCRIPTION
**Pull request by Andras-Csanyi**
## PR summary

Changes in this PR are going to 
- change `ibm-platform-services` package name to `platform-services`, and 
- move it to `@ibm-cloud` organisation
- update README according to the above changes
 
From this point forward the package is `@ibm-cloud/platform-services`.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,26 +47,26 @@ IBM Cloud services:
...
-[IAM Policy Management](https://cloud.ibm.com/apidocs/iam-policy-management) | ibm-platform-services/iam-policy-management/v1
-[IBM Cloud Shell](https://cloud.ibm.com/apidocs/cloudshell) | ibm-platform-services/ibm-cloud-shell/v1
-[Open Service Broker](https://cloud.ibm.com/apidocs/resource-controller/ibm-cloud-osb-api) | ibm-platform-services/open-service-broker/v1
-[Posture Management](https://cloud.ibm.com/apidocs/security-compliance/posture) | ibm-platform-services/posture-management/v1
-[Resource Controller](https://cloud.ibm.com/apidocs/resource-controller/resource-controller) | ibm-platform-services/resource-controller/v2
-[Resource Manager](https://cloud.ibm.com/apidocs/resource-controller/resource-manager) | ibm-platform-services/resource-manager/v2
-[Usage Metering](https://cloud.ibm.com/apidocs/usage-metering) | ibm-platform-services/usage-metering/v4
-[Usage Reports](https://cloud.ibm.com/apidocs/metering-reporting) | ibm-platform-services/usage-reports/v4
-[User Management](https://cloud.ibm.com/apidocs/user-management) | ibm-platform-services/user-management/v1
+[Case Management](https://cloud.ibm.com/apidocs/case-management) | platform-services/case-management/v1
```
Comment (padamstx) [inline README.md:50]: @dpopp07 can confirm this, but I'm pretty sure that the `@ibm-cloud/` part is needed in an import path.

**Review by padamstx — CHANGES_REQUESTED** [review]
Within the README (list of services) I think we need to include `@ibm-cloud/`in the import path.  Pretty sure an app would need to use something like this to import the package:
```
   const CaseManagementV1 = require(`@ibm-cloud/platform-services/case-management/v1`);
```

**Review by padamstx — APPROVED** [review]
LGTM

**Review by dpopp07 — APPROVED** [review]
Looks good! 👍

**Pull request merged**